### DISCUSSION TITLE
The community is keeping this game alive.

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1882] [General]
The game would have died months ago if it weren't for the amount of community creations. It's been months since a new week was released, and more since this repo has got a new week.

Please, do something to keep this game alive. The mods won't go on forever.

**Comment by CubixL** [comment]
i can wait another 5 years for the full ass game to release

**Reply by ghost** [reply]
we havent got a single new week outta this yet rememeebrrbekrer
